src/__tests__/regression/test-cases/read-state.json

```diff
@@ -95,13 +94,9 @@
   "8zGVX17V6u3Uzn2fWJFya3vEuPOvIZ6FRifUDhaprtE",
   "Zwp7r7Z10O0TuF6lmFApB7m5lJIrE5RbLAVWg_WKNcU",
   "1enYq8LDJQ7fxW20WMWxbq3jpMaBeN-k67WXnhEDIds",
-  "efrXSmKuDH32Svllu9g15PxTlxrL5T3UNT1jxgQqat0",
```

Comment:
all those contracts were using unsafeClient to load tx data from arweave and were causing random test errors.